test/card_example_dataABCD.yml

```diff
@@ -3,17 +3,17 @@ campaign:
   crab_template: template_crab.py
 
   # User specific
-  workArea: data22_pub_yml # New each time
+  workArea: data22abcd_pub_yml # New each time
   storageSite: T2_DE_RWTH # Make sure you have write access
   outLFNDirBase: /store/user/anstein/PFNano # Change username and path
   voGroup: dcms # or leave empty
 
   # Campaign specific
-  tag_extension: BTV_Run3_2022_Comm_v1 # Will get appended after the current tag
+  tag_extension: BTV_Run3_2022_Comm_v2 # Will get appended after the current tag
   tag_mod: # Will modify name in-place for MC eg. "PFNanoAODv1" will replace MiniAODv2 -> PFNanoAODv1
   # If others shall be able to access dataset via DAS (important when collaborating for commissioning!)
   publication: True
-  config: nano_data_2022_NANO.py
```

Comment:
We could then delete the PromptReco config (nano_data_2022_NANO.py) from this branch, right? As it's not run anymore in any card.

Reply:
Even if we take into account that for 2022FG there will be PromptReco in nano v11 and not yet re-reco, we will anyway need another config with new GT for that. So the old nano_data_2022_NANO.py will be outdated in any case and can be removed.

Reply:
My bad, I forgot to delete the old config, and forgot to replace it in the yml file with the re-reco one!